### ISSUE TITLE
Test function shouldn't be available in normal configuration

### ISSUE THREAD

**Issue by ZerothLaw** [pi-hole/api#57]
https://github.com/pi-hole/api/blob/cf5ff3cdcf9df156cc36ba625095a91378f22f9d/src/setup.rs#L63

A test function like this should have `#[cfg(test)]` set on it - it shouldn't be usable from normal configurations.

**Comment by AzureMarker** [comment]
This is fixed in [`feature/shared-memory`](https://github.com/pi-hole/api/blob/9a19360eea23c07c10e495b00c42c2e5966906b0/src/setup.rs#L71)

**Comment by ZerothLaw** [comment]
Just reviewing the main branch. Also found this: 
https://github.com/pi-hole/api/blob/master/src/ftl.rs#L36

**Comment by ZerothLaw** [comment]
Ah, I see already found and fixed in that branch. 

If you'd like, I'll review that branch instead?

**Comment by AzureMarker** [comment]
Yes, that branch is the latest code and will be merged into `development` once it is complete. It's a large change (shared memory), so it is living in that branch instead of `development` for now.

**Comment by AzureMarker** [comment]
Thank you for reviewing the code though! It has been a journey getting this API to a state where it can be shared with the community.

**Comment by ZerothLaw** [comment]
Happy to help. It's a cool project and I totally understand that,
